src/components/workshops/CarouselSubmissionForm.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component, Fragment } from 'react'
+import api from 'api'
+import { isURL } from 'validator'
+import styled from 'styled-components'
+import Auth from 'components/Auth'
+import {
+  Box,
+  Flex,
+  Label,
+  Heading,
+  Text,
+  Button,
+  Field,
+  theme,
+} from '@hackclub/design-system'
+
+const CarouselSubmissionFormOuter = styled(Flex).attrs({
+  p: [2, 3, 4],
+  bg: theme.colors.white,
+  align: 'center',
+})`
+  border-radius: 5px;
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-direction: column;
+  ${theme.mediaQueries.md} {
+    border-radius: 10px;
+  }
+`
+
+const LiveField = styled(Field).attrs({
+  label: 'Live URL',
+  name: 'Live URL',
+  placeholder: "(where's the final product?)",
```

Comment:
Always use rich quotes for copy, & single quotes (or template strings) in code:
```suggestion
  placeholder: '(where’s the final product?)',
```